app.js

```diff
@@ -166,7 +164,7 @@ app.get('*', function (request, response) {
   } else if (request.path === "/get-house-events") {
     getHouseEvents(response);
   } else {
-    response.sendfile(path.join(__dirname, '/public/index.html'));
+    response.sendFile(path.join(__dirname, '/public/index.html'));
```

Comment:
https://github.com/expressjs/express/wiki/Migrating-from-3.x-to-4.x#deprecated-since-3x